flocx_market/tests/unit/api/test_app_bid.py

```diff
@@ -19,7 +19,7 @@
     end_time=now,
     duration=16400,
     status="available",
-    server_config_query={'foo': 'bar'},
+    server_config_query={'specs': 'bar'},
```

Comment:
@ljmcgann I'm a little confused; shouldn't `server_config_query` be a list (of tuples)? Maybe I missed some discussion when I was on vacation or something...